apps/admin-be/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "tsconfig/node.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "include": ["src/**/*"]
+}
```

Comment:
We should be extending the common tsconfig - refer to the other apps to see how we're doing that.

Reply:
Line 2 means we are extending from the shared tsconfig no?